web/src/package.json

```diff
@@ -136,6 +136,7 @@
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-typescript-to-proptypes": "1.2.1",
     "chance": "1.1.4",
+    "copy-webpack-plugin": "5.1.1",
```

Comment:
I'm not seeing any other references to `html-webpack-plugin`, so we should be good to remove it here